tensorboard/plugins/interactive_inference/witwidget/notebook/base.py

```diff
@@ -16,10 +16,10 @@
 import json
 import googleapiclient.discovery
 import tensorflow as tf
-import sys
 from IPython import display
 from google.protobuf import json_format
 from numbers import Number
+from six import ensure_str
```

Comment:
You may want to add six dependency on the BUILD file.

Reply:
done